guides/release/tutorial/deploying.md

```diff
@@ -32,6 +32,34 @@ You can deploy your application to any web server by copying the output from `em
 scp -r dist/* myserver.com:/var/www/public/
 ```
 
+## Deploying to ZEIT Now
+
+[ZEIT Now](https://zeit.co) is a cloud platform for websites and serverless APIs, that you can use to deploy your Ember projects to your personal domain (or a free `.now.sh` suffixed URL).
+
+This guide will show you how to get started in a few quick steps:
+
+### Step 1: Installing Now CLI
+
+To install their command-line interface with [npm](https://www.npmjs.com/package/now), run the following command:
+
+```bash
+npm install -g now
+```
+
+### Step 2: Deploying
+
+You can deploy your application by running the following command in the root of the project directory:
+
+```bash
+now
+```
+
+**Alternatively**, you can also use their integration for [GitHub](https://zeit.co/github) or [GitLab](https://zeit.co/gitlab).
+
+That’s all!
+
+Your site will deploy, and you will receive a link similar to the following: https://ember.now-examples.now.sh
+
 ## Deploying to surge.sh
```

Comment:
- Change from a level 2 heading to a level 3 heading
- change text to "Surge"

Reply:
Done